tst/testinstall/semigrp.tst

```diff
@@ -477,7 +477,7 @@ true
 # test the methods for Random
 gap> S := FreeSemigroup(3);;
 gap> Random(S);
-s3*s1
+s1*s2
```

Comment:
This test change perhaps was made necessary by those other accidental changes?

Reply:
I don't think its the accidental changes but that an fp group created earlier from a finite group now immediately knew a faithful rep. and did not have to construct it.

In any case, tests for explicit results of `Random` should be expected to be highly unstable.

Reply:
Sure.